.github/ISSUE_TEMPLATE/release.md

```diff
@@ -16,6 +16,8 @@ about: Create release checklist
 - [ ] update downstream backport of [openshift/kube-rbac-proxy](https://github.com/openshift/kube-rbac-proxy)
 - [ ] update downstream backport of [openshift/k8s-prometheus-adapter](https://github.com/openshift/k8s-prometheus-adapter)
 - [ ] update downstream backport of [openshift/grafana](https://github.com/openshift/grafana)
+- [ ] update downstream backport of [openshift/prom-label-proxy](https://github.com/openshift/prom-label-proxy)
+- [ ] update downstream backport of [openshift/procfs](https://github.com/openshift/procfs)
```

Comment:
This repository is a dependency of `github.com/openshift/node_exporter` so it should probably go with the `node_exporter` instructions. The workflow would be:
1. Bump `openshift/procfs` to the version used in the node_exporter's upstream version we want to bump to, including any downstream patch we may have.
2. Bump `openshift/node_exporter` to the upstream version replacing the `procfs` dependency by the downstream repo.